core/tooling/src/main/scala/cli/CodepathCoverageMain.scala

```diff
@@ -46,6 +46,10 @@ object CodepathCoverageMain extends App {
   }
   preparationClient.resourceList(sparkPvc).inNamespace("swiss-knife").createOrReplace()
 
+  val microservicesCRD =
```

Comment:
There is also a cloudflow crd now you could maybe use?

Reply:
We should use both to ensure coverage.